js1/uvod-do-js/prvni-program.md

```diff
@@ -13,18 +13,17 @@ Naše JavaScriptové programy budou vždy součástí nějaké webové stránky.
     <title>První program</title>
   </head>
   <body>
-    <h1>První program</h1>
   </body>
 </html>
 ```
 
-Pro náš JavaScriptový kód budeme vždy vytvářet oddělený soubor, podobně jako jste zvyklí vytvářet oddělený soubor pro CSS styly. JavaScriptovému programu většinou budeme dávat název `index.js`. Jeho obsah může vypadat například takto.
+Když tuto stránku zobrazíte v prohlížeči, uvidíte pouze prázdné bílé okno. Ideální inspirace pro naši první JavaScriptovou aplikaci. Založme si soubor `index.js` a vložme do něj následující příkaz.
 
 ```js
-document.body.innerHTML = '<h1>ahoj</h1>';
+document.body.innerHTML += 'Vítejte na našem webu';
```

Comment:
Když už tady dáváš rovnou `+=`, tak bych tu `<h1>` v HTML spíš nechal.

S H1 se lépe pozná, že účastnice si otevřela server ve správné složce a mně se třeba lépe vysvětluje, co to `+=` dělá.

Reply:
💟

Reply:
> měl byste vidět nadpis

Jen teda tenhle text je potřeba taky vrátit, ne? (A ještě v něm teda byl překlep teď koukám.)

Reply:
Opraveno